.golangci.yml

```diff
@@ -103,3 +102,10 @@ issues:
         - golint
         - deadcode
         - godot
+
+linters-settings:
+  lll:
+    line-length: 256
+  funlen:
```

Comment:
I think we can disable this check. In the following pr, we need to first enable that with high priority. 

As to the priority, If we have no enough time to analyze I suggest looking at https://github.com/istio/istio/blob/494aca43715d2f970bdb76ce1dfcc5e1095f3070/common/config/.golangci.yml#L34-L53

Reply:
OK